src/index.ts

```diff
@@ -45,6 +45,27 @@ cmdClient.addEvents(events);
   // client has received the READY payload, do stuff now
   console.log(`Online with ${client.shardCount} shards`);
   cmdClient.ready = true;
+
+  // Add owners from config
+  config.owners.forEach(uid => {
+    const user = shardClient.users.get(uid);
+    if (!user) {
+      /*
+       * Try to fetch from the API
+       * TODO: fetching from API the best option here? perhaps make a fake user and fill in later
+       * Or maybe get from the gateway? a few options here...
+       */
+      shardClient.rest.fetchUser(uid).then(apiUser => {
```

Comment:
Why do we need to fetch the users? Seems like a needless request when the config holds the userid already. Why not compare if the config includes the message author?